packages/react/src/components/AnimatedHeader/__stories__/AnimatedHeader.mdx

```diff
@@ -93,7 +93,7 @@ In your styles file import
 
 ```
 @use '@carbon/react';
-@use '@carbon-labs/react-animated-header/scss/animated-header';
+@use '@carbon-labs/react-animated-header/scss/animated_header';
```

Comment:
This was a typo.. should be set back to `animated-header` and then change the filename (packages/react/src/components/AnimatedHeader/components) to use '-' instead of '_' to stay consistent with other style files